draft-ietf-dprive-dnsoquic.md

```diff
@@ -585,7 +585,7 @@ to the following recommendation:
 * if padding at the QUIC level is not available or not used,
   DNS over QUIC MUST ensure that all DNS queries and responses are padded to
   a small set of fixed sizes, using the EDNS padding extension as specified
-  in "Padding Policies for Extension
+  in {{!RFC7830}}. The sizes SHOULD be aligned with the recommendations of the "Padding Policies for Extension
   Mechanisms for DNS (EDNS(0))" {{!RFC8467}}.
 
```

Comment:
RFC8467 is experimental, so the exclamation mark and the capital SHOULD trigger a down reference. Can we express the same idea differently, such as "In the absence of standard policy for padding, implementers may consider following the recommendations of "Padding Policies for Extension Mechanisms for DNS (EDNS(0))" {{?RFC8467}}."

Reply:
Based on Brians email of 7th December (and Eric's follow up) I thought we were OK to keep this as normative for now:

> RFC 3967 allows for waivers for down references, but they need to be called out during IETF Last Call. 
My intent is to indicate in the shepherd writeup that RFC 8094 can be moved to an Informative reference, 
but that a down-ref waiver is being requested for RFC 8467.

But your suggested text works too, and I see now we can move that recommendation out of the bullets because it is the same for both cases. How about using a lower case should as a middle ground?

> * if the implementation of QUIC exposes APIs to set a padding policy,
  DNS over QUIC SHOULD use that API to align the packet length to a small set of fixed sizes.

> * if padding at the QUIC level is not available or not used,
  DNS over QUIC MUST ensure that all DNS queries and responses are padded to
  a small set of fixed sizes.

> In the absence of a standard policy for padding sizes, implementations should consider following the recommendations of the Experimental status "Padding Policies for Extension Mechanisms for DNS (EDNS(0))" {{?RFC8467}}.

Reply:
Yes, that works for me.

Reply:
@alex-nicat  How does this look now?